### DISCUSSION TITLE
ACTIVITIES HOOK => ACCOUNTABILITY

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#17208] [Feature Requests]
Hi everyone,
it would be great if the ACCOUNTABILITY object could be passed through to any ACTIVITIES record creations.

Reason: I want to log custom claims / custom accountability in the activities log... and currently thats not really possible ( or just via ugly workarounds which don't work in all cases)

Thx
Roman

**Comment by ernst86913** [comment]
PS - would also be great for the revisions.

**Comment by paescuj** [comment]
> [...] I want to log custom claims / custom accountability [...]

That means you'd like to be able to modify the object?

**Reply by ernst86913** [reply]
I already modified the claims / accountability objects using the hooks that are existing in Directus.... thats really great.

Now I added a field in the ACTIVITY / REVISIONS table... and with every new activity / revision I have a hook which would then take the field from the accountability and add it to activity / revisions... 

Especially for ACTIVITY its obvious that accountability should be past through... because if you have custom claims you want to log those in the activity....

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
